components/NavItems/NavItems.js

```diff
@@ -0,0 +1,109 @@
+class NavItems {
+  render() {
+    let htmlCatalog = "";
+    NAV_ITEMS.forEach(({title, submenu}) => {
+      if(submenu.length !== 0){
+        htmlCatalog += `
+        <li class="nav-item">
+          <a href="#" class="menu-link-has-submenu">${title}</a>
+          <span class="menu-arrow"></span>
+          <ul class="menu-sublist">
+        `
+        submenu.forEach((item) => {
+          switch(item){
```

Comment:
Do you really need switch case here? It's a rare construction that you can usually avoid.